src/rgw/rgw_lc.cc

```diff
@@ -774,7 +774,7 @@ class LCOpAction_NonCurrentExpiration : public LCOpAction {
     auto& o = oc.o;
     int r = remove_expired_obj(oc, true);
     if (r < 0) {
-      ldout(oc.cct, 0) << "ERROR: remove_expired_obj " << dendl;
+      ldout(oc.cct, 0) << "ERROR: remove_expired_obj " << oc.bucket_info.bucket << ":" << o.key << "error code: " << std::to_string(r) << dendl;
```

Comment:
can you consider wrapping this line, also strerror instead of just stringifying error code

Reply:
1) How have you thought to wrap it? 
2) I'll use r to get the error.

Thank you

Reply:
@ofriedma i think we wrap lines at 80 chars, so something like:
```c++
ldout(oc.cct, 0) << "ERROR: remove_expired_obj "
                 << oc.bucket_info.bucket << ":" << o.key
                 << "(" << cpp_strerror(r) << ")" << dendl; ```